pkg/cmd/server/api/v1/conversions.go

```diff
@@ -93,6 +98,56 @@ func init() {
 			out.KeyFile = in.ServerCert.KeyFile
 			return nil
 		},
+		func(in *NodeConfig, out *newer.NodeConfig, s conversion.Scope) error {
```

Comment:
If we're going to have conversions, you'll need to add a fuzzer test.  We were dodging it before because only small types were being converted, but converting the entire `NodeConfig` via conversion function is worthy of fuzzing.

Checkout https://github.com/openshift/origin/blob/master/pkg/api/serialization_test.go